vital/types/metadata_tags.h

```diff
@@ -48,95 +48,93 @@
 //      tag                          string name                        type
 //      ---                          -----------                        ----
 #define KWIVER_VITAL_METADATA_TAGS(CALL)                                \
```

Comment:
It bugs me there are no units here, not sure if that is documented somewhere else...

Reply:
I could add a more extended description to some of these items (and risk the wrath of the line length enforcers). Additional description could also be helpful for some items. I could add an additional field with an extended description.